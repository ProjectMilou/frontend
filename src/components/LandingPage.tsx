import React from 'react';
import Box from '@material-ui/core/Box';
import Container from '@material-ui/core/Container';
import CardMedia from '@material-ui/core/CardMedia';

const LandingPage: React.FC = () => (
<Box component="span">
<Container fixed>
<CardMedia>
  <img src="https://getmilou.de/wp-content/uploads/elementor/thumbs/2020-12-15-Logo-hell-p17eq5ztwznwc003uog4y2jecjf0xlb7ut5p5vmfus.png" title="2020-12-15 Logo hell" alt="2020-12-15 Logo hell"/>
</CardMedia>
<div className="elementor-widget-container">
  <h1 className="elementor-heading-title elementor-size-xxl">
    <b>Empower to invest.</b>
  </h1>
</div>
</Container>
</Box>
);
export default LandingPage;