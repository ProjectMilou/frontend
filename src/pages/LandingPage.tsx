import React from 'react';
import { RouteComponentProps } from '@reach/router';

import {
  Button,
  Box,
  Grid,
  Typography,
  Divider,
  makeStyles,
  Theme,
  createStyles,
} from '@material-ui/core';

import MoneyIcon from '@material-ui/icons/Money';
import LoyaltyIcon from '@material-ui/icons/Loyalty';

import analyse from '../assets/images/analyse.png';
import lernen from '../assets/images/lernen.png';
import portfolio from '../assets/images/Portfolio.png';
import LogoCard from '../components/shell/landingPage/LogoCard';
import Card from '../components/shell/landingPage/Card';
import GridRowContainer from '../components/shell/landingPage/GridRowContainer';
import InfoCard from '../components/shell/landingPage/InfoCard';

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    mediaSmall: {
      maxWidth: '450px',
      maxHeight: '250px',
      display: 'auto',
      objectFit: 'scale-down',
      padding: '20px',
    },
    mediaLarge: {
      width: '500px',
      height: '732px',
      padding: '50px',
      objectFit: 'scale-down',
    },
    icon: {
      width: '20px',
      height: '20px',
    },
    divider: {
      light: false,
      variant: 'middle',
      marginLeft: '250px',
      marginRight: '250px',
      height: '1px',
      background: theme.palette.secondary.contrastText,
    },
    box1: {
      paddingInline: '50px',
      paddingTop: '50px',
      paddingBottom: '50px',
    },
    typographyMain: {
      color: theme.palette.primary.dark,
      fontSize: '24px',
      marginLeft: '120px',
      marginRight: '120px',
    },
    boxMain: {
      paddingInline: '50px',
      marginBlock: '100px',
      backgroundColor: theme.palette.primary.contrastText,
    },
    gridItem: {
      paddingInline: '0px',
    },
    smallIcon: {
      fontSize: '54px',
      color: theme.palette.secondary.light,
    },
    span: {
      color: theme.palette.secondary.light,
    },
    success: {
      color: theme.palette.success.main,
    },
    warning: {
      color: theme.palette.warning.main,
    },
    error: {
      color: theme.palette.error.main,
    },
    box: {
      backgroundColor: theme.palette.primary.main,
      padding: '80px',
      marginBottom: '60px',
      display: 'flex',
      justifyContent: 'center',
    },
    typography: {
      color: '#50aaff',
      fontWeight: 'bold',
      padding: '20px',
    },
    button: {
      backgroundColor: theme.palette.secondary.light,
      borderRadius: '10px',
      color: theme.palette.primary.contrastText,
      textTransform: 'none',
    },
    title: {
      fontWeight: 'bold',
      padding: '16px',
    },
  })
);

const LandingPage: React.FC<RouteComponentProps> = () => {
  const classes = useStyles();
  return (
    <Box>
      <Card />
      <Divider className={classes.divider} />
      <Box className={classes.box1}>
        <Typography
          variant="body1"
          align="center"
          className={classes.typographyMain}
        >
          Apps like Trade Republic and other direct brokers make investing
          easier and easier. The stock market is now accessible to everyone.
          Even so, many individuals do not have access to tools to make informed
          and informed investment decisions. Current solutions are often
          confusing and tailored to professional investors. This is exactly
          where we attack.
        </Typography>
      </Box>
      <Divider className={classes.divider} />
      <Box className={classes.boxMain}>
        <Grid
          container
          direction="column"
          justify-content="center"
          justify="center"
          alignItems="center"
          spacing={10}
        >
          <GridRowContainer
            image={lernen}
            title="LEARN"
            content="No prior knowledge is necessary for us. Whether you are a professional or a beginner, we will accompany you throughout the entire investment lifecycle and help you to be a successful investor. With our free academy, you can easily build up your knowledge of finances and deepen it."
            classNameImage={classes.mediaSmall}
            classNameGrid={classes.gridItem}
            type="even"
            titleClass={`${classes.warning} ${classes.title}`}
          />
          <GridRowContainer
            image={analyse}
            title="ANALYSIS"
            content="Do you feel ready for your first investment or are you interested in the development of different stocks? Perfect! Our analysis tool helps you to conduct investment research. Regardless of whether you want to invest in stocks or ETFs: Here you will find the most important things clearly and understandably!"
            classNameImage={classes.mediaSmall}
            classNameGrid={classes.gridItem}
            type="odd"
            titleClass={`${classes.error} ${classes.title}`}
          />

          <GridRowContainer
            image={portfolio}
            title="PORTFOLIO"
            content="It is also important to us that you build your portfolio with a good risk-return ratio. Our tool helps you to achieve optimal diversification in order to reduce your risk. In doing so, we are guided by the current scientific standards of portfolio theory."
            classNameImage={classes.mediaSmall}
            classNameGrid={classes.gridItem}
            type="even"
            titleClass={`${classes.success} ${classes.title}`}
          />
        </Grid>
      </Box>

      <Box bgcolor="#0c1a3a" color="white" px={20} pt={4} pb={18}>
        <Grid container direction="column" spacing={2}>
          <Grid item spacing={2}>
            <Typography variant="h3">Did you know...</Typography>
          </Grid>
          <Grid item container direction="row" spacing={2} alignItems="center">
            <Grid item>
              <MoneyIcon className={classes.smallIcon} />
            </Grid>
            <Grid item xs>
              <Typography variant="h6">
                ..to be a successful investor, initially, you{' '}
                <span className={classes.span}>need no assets</span> . You can
                start investing with as little as 25 euros a month.
              </Typography>
            </Grid>
          </Grid>
          <Grid item container direction="row" spacing={2} alignItems="center">
            <Grid item>
              <LoyaltyIcon className={classes.smallIcon} />
            </Grid>
            <Grid item xs>
              <Typography variant="h6">
                ..we only work with{' '}
                <span className={classes.span}>commission-free products</span>,
                stocks and ETFs. This ensures that we pursue the{' '}
                <span className={classes.span}>same interests </span>
                as you, namely to make you a successful investor.
              </Typography>
            </Grid>
          </Grid>
          <Grid item alignItems="center">
            <Typography
              variant="h3"
              align="center"
              style={{ fontWeight: 'bold', marginTop: '64px' }}
            >
              Milou accompanies you on all your financial adventures.
            </Typography>
          </Grid>
          <Grid item container direction="column">
            <Typography
              variant="h6"
              align="center"
              style={{ marginTop: '16px', marginBottom: '32px' }}
            >
              We help you to make well-founded investment decisions yourself.
            </Typography>
            <div style={{ margin: 'auto' }}>
              <Button
                href="/analyser"
                style={{
                  backgroundColor: '#FFC43B',
                  color: 'white',
                }}
              >
                Start for free
              </Button>
            </div>
          </Grid>
        </Grid>
      </Box>

      <InfoCard />
      <LogoCard />
    </Box>
  );
};

export default LandingPage;
