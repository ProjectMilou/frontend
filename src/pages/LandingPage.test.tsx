import { fireEvent, render, screen, waitFor } from '@testing-library/react';
import * as React from 'react';
import LandingPage from './LandingPage';


describe('LandingPage', () => {
  test('should render landingPage', async () => {
    const { container } = render(<LandingPage />);
    expect(container).toMatchSnapshot();
  });

  test('should navigate to analyser', async () => {
    render(<LandingPage />);
    const goToAnalyser1 = screen.getByText(/Start now/i);
    expect(goToAnalyser1).toBeInTheDocument();
    fireEvent.click(goToAnalyser1);
    waitFor(() => expect(window.location.pathname).toBe('/analyser')).then(() => {});

    window.location.pathname = '/';
    expect(window.location.pathname).toBe('/')

    const goToAnalyser2 = screen.getByText(/Start for free/i);
    expect(goToAnalyser2).toBeInTheDocument();
    fireEvent.click(goToAnalyser2);
    waitFor(() => expect(window.location.pathname).toBe('/analyser')).then(() => {});
  });
});