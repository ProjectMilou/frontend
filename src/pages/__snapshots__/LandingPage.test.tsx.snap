// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should render landingPage 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-21 makeStyles-box-18"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-wrap-xs-nowrap MuiGrid-align-items-xs-center"
    >
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <img
          alt="2020-12-15 Logo hell"
          src="milou.png"
          title="2020-12-15 Logo hell"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <h2
          class="MuiTypography-root makeStyles-typography-19 MuiTypography-h2"
        >
          Empower to invest.
        </h2>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <a
          aria-disabled="false"
          class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-20"
          href="/analyser"
          tabindex="0"
        >
          <span
            class="MuiButton-label"
          >
            Start now
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
      </div>
    </div>
  </div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-content-xs-center MuiGrid-justify-xs-center"
  >
    <div
      class="MuiGrid-root MuiGrid-item"
    >
      <hr
        class="MuiDivider-root makeStyles-divider-4"
      />
    </div>
    <div
      class="MuiGrid-root MuiGrid-item"
    >
      <p
        class="MuiTypography-root makeStyles-typographyMain-6 MuiTypography-body1 MuiTypography-alignCenter"
      >
        Apps like Trade Republic and other direct brokers make investing easier and easier. The stock market is now accessible to everyone. Even so, many individuals do not have access to tools to make informed and informed investment decisions. Current solutions are often confusing and tailored to professional investors. This is exactly where we attack.
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item"
    >
      <hr
        class="MuiDivider-root makeStyles-divider-4"
      />
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root-22 makeStyles-boxMain-7"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      justify-content="center"
    >
      <div
        class="MuiBox-root MuiBox-root-23"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
          justify-content="space-between"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
            style="width: 500px; height: 360px;"
          >
            <img
              class="MuiCardMedia-root makeStyles-mediaSmall-1 MuiCardMedia-media MuiCardMedia-img"
              src="lernen.png"
            />
          </div>
          <div
            class="MuiGrid-root MuiGrid-item"
            style="width: 500px; height: 360px;"
          >
            <div
              class="MuiGrid-root makeStyles-gridItem-8 MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-true"
              style="width: 500px;"
            >
              <div
                class="makeStyles-warning-12 makeStyles-title-17"
              >
                <h4
                  class="MuiTypography-root makeStyles-warning-12 makeStyles-title-17 MuiTypography-h4"
                >
                  LEARN
                </h4>
              </div>
              <div
                style="max-width: 550px;"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
                >
                  No prior knowledge is necessary for us. Whether you are a professional or a beginner, we will accompany you throughout the entire investment lifecycle and help you to be a successful investor. With our free academy, you can easily build up your knowledge of finances and deepen it.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root MuiBox-root-24"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
          justify-content="space-between"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
            style="width: 500px; height: 360px;"
          >
            <div
              class="MuiGrid-root makeStyles-gridItem-8 MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-true"
              style="width: 500px;"
            >
              <div
                class="makeStyles-error-13 makeStyles-title-17"
              >
                <h4
                  class="MuiTypography-root makeStyles-error-13 makeStyles-title-17 MuiTypography-h4"
                >
                  ANALYSIS
                </h4>
              </div>
              <div
                style="max-width: 550px;"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
                >
                  Do you feel ready for your first investment or are you interested in the development of different stocks? Perfect! Our analysis tool helps you to conduct investment research. Regardless of whether you want to invest in stocks or ETFs: Here you will find the most important things clearly and understandably!
                </p>
              </div>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item"
            style="width: 500px; height: 360px;"
          >
            <img
              class="MuiCardMedia-root makeStyles-mediaSmall-1 MuiCardMedia-media MuiCardMedia-img"
              src="analyse.png"
            />
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root MuiBox-root-25"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
          justify-content="space-between"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
            style="width: 500px; height: 360px;"
          >
            <img
              class="MuiCardMedia-root makeStyles-mediaSmall-1 MuiCardMedia-media MuiCardMedia-img"
              src="Portfolio.png"
            />
          </div>
          <div
            class="MuiGrid-root MuiGrid-item"
            style="width: 500px; height: 360px;"
          >
            <div
              class="MuiGrid-root makeStyles-gridItem-8 MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-true"
              style="width: 500px;"
            >
              <div
                class="makeStyles-success-11 makeStyles-title-17"
              >
                <h4
                  class="MuiTypography-root makeStyles-success-11 makeStyles-title-17 MuiTypography-h4"
                >
                  PORTFOLIO
                </h4>
              </div>
              <div
                style="max-width: 550px;"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
                >
                  It is also important to us that you build your portfolio with a good risk-return ratio. Our tool helps you to achieve optimal diversification in order to reduce your risk. In doing so, we are guided by the current scientific standards of portfolio theory.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root-26"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-item"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3"
            style="padding: 8px;"
          >
            Did you know...
          </h3>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-item MuiGrid-align-items-xs-center"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-smallIcon-9"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M5 8h2v8H5zm7 0H9c-.55 0-1 .45-1 1v6c0 .55.45 1 1 1h3c.55 0 1-.45 1-1V9c0-.55-.45-1-1-1zm-1 6h-1v-4h1v4zm7-6h-3c-.55 0-1 .45-1 1v6c0 .55.45 1 1 1h3c.55 0 1-.45 1-1V9c0-.55-.45-1-1-1zm-1 6h-1v-4h1v4z"
              />
              <path
                d="M2 4v16h20V4H2zm2 14V6h16v12H4z"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              ..to be a successful investor, initially, you
               
              <span
                class="makeStyles-span-10"
              >
                need no assets
              </span>
               . You can start investing with as little as 25 euros a month.
            </h6>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-item MuiGrid-align-items-xs-center"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-smallIcon-9"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M21.41 11.58l-9-9C12.05 2.22 11.55 2 11 2H4c-1.1 0-2 .9-2 2v7c0 .55.22 1.05.59 1.42l9 9c.36.36.86.58 1.41.58.55 0 1.05-.22 1.41-.59l7-7c.37-.36.59-.86.59-1.41 0-.55-.23-1.06-.59-1.42zM5.5 7C4.67 7 4 6.33 4 5.5S4.67 4 5.5 4 7 4.67 7 5.5 6.33 7 5.5 7zm11.77 8.27L13 19.54l-4.27-4.27C8.28 14.81 8 14.19 8 13.5c0-1.38 1.12-2.5 2.5-2.5.69 0 1.32.28 1.77.74l.73.72.73-.73c.45-.45 1.08-.73 1.77-.73 1.38 0 2.5 1.12 2.5 2.5 0 .69-.28 1.32-.73 1.77z"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-true"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              ..we only work with
               
              <span
                class="makeStyles-span-10"
              >
                commission-free products
              </span>
              , stocks and ETFs. This ensures that we pursue the
               
              <span
                class="makeStyles-span-10"
              >
                same interests 
              </span>
              as you, namely to make you a successful investor.
            </h6>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center"
        >
          <h3
            class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter"
            style="font-weight: bold; margin-top: 64px;"
          >
            Milou accompanies you on all your financial adventures.
          </h3>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column"
        >
          <h6
            class="MuiTypography-root MuiTypography-h6 MuiTypography-alignCenter"
            style="margin-top: 16px; margin-bottom: 32px;"
          >
            We help you to make well-founded investment decisions yourself.
          </h6>
          <div
            style="margin: auto;"
          >
            <a
              aria-disabled="false"
              class="MuiButtonBase-root MuiButton-root MuiButton-text"
              href="/analyser"
              style="background-color: rgb(255, 196, 59); color: white;"
              tabindex="0"
            >
              <span
                class="MuiButton-label"
              >
                Start for free
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root-35 makeStyles-boxMain-29"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      justify-content="center"
    >
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <img
          class="MuiCardMedia-root makeStyles-mediaLarge-27 MuiCardMedia-media MuiCardMedia-img"
          src="media2.png"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-grid-xs-true"
        justify-content="center"
        style="max-width: 600px;"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-align-items-xs-center"
          style="width: 500px; height: 180px;"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-success-32 makeStyles-largeIcon-30"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M42 6H6c-2.2 0-4 1.8-4 4v6h4v-6h36v28H28v4h14c2.2 0 4-1.8 4-4V10c0-2.2-1.8-4-4-4zM2 36v6h6c0-3.32-2.68-6-6-6zm0-8v4c5.52 0 10 4.48 10 10h4c0-7.74-6.26-14-14-14zm0-8v4c9.94 0 18 8.06 18 18h4c0-12.16-9.86-22-22-22zm20 2.18v4L29 30l7-3.82v-4L29 26l-7-3.82zM29 12l-11 6 11 6 11-6-11-6z"
                transform="scale(0.5, 0.5)"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-true"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5"
            >
              LEARNING PLATFORM
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              Our learning journey picks you up where you are. With our short tutorials you can acquire new knowledge within a very short time and thus become an investor yourself.
            </p>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-align-items-xs-center"
          style="width: 500px; height: 180px;"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-warning-33 makeStyles-largeIcon-30"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-true"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5"
            >
              INFO BOXES
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              With the help of our info boxes, you can also view explanations of key figures and values in the app at any time and acquire knowledge.
            </p>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-align-items-xs-center"
          style="width: 500px; height: 180px;"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-span-31 makeStyles-largeIcon-30"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16 6l2.29 2.29-4.88 4.88-4-4L2 16.59 3.41 18l6-6 4 4 6.3-6.29L22 12V6z"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-true"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5"
            >
              SIMPLE VISUALIZATION
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              With the help of simple graphics and diagrams, you get information at a glance and can draw conclusions about individual stocks and your portfolio.
            </p>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-align-items-xs-center"
          style="width: 500px; height: 180px;"
        >
          <div
            class="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-error-34 makeStyles-largeIcon-30"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
              />
            </svg>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-true"
          >
            <h5
              class="MuiTypography-root MuiTypography-h5"
            >
              RISK WARNING
            </h5>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              Identify risks across your entire portfolio with simple warnings and learn how to counteract them.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root-37"
  >
    <div
      class="MuiGrid-root MuiGrid-container"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-grid-xs-true"
      >
        <h3
          class="MuiTypography-root MuiTypography-h3"
        >
          Our supporters
        </h3>
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
        >
          Thank you for your help!
        </p>
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-38"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
      >
        <ul
          class="MuiGridList-root"
          style="margin: -9px;"
        >
          <li
            class="MuiGridListTile-root"
            style="width: 16.666666666666668%; height: 138px; padding: 9px;"
          >
            <div
              class="MuiGridListTile-tile"
            >
              <img
                class="MuiCardMedia-root makeStyles-logo-36 MuiCardMedia-media MuiCardMedia-img"
                src="180.png"
              />
            </div>
          </li>
          <li
            class="MuiGridListTile-root"
            style="width: 16.666666666666668%; height: 138px; padding: 9px;"
          >
            <div
              class="MuiGridListTile-tile"
            >
              <img
                class="MuiCardMedia-root makeStyles-logo-36 MuiCardMedia-media MuiCardMedia-img"
                src="getThumb.gif"
              />
            </div>
          </li>
          <li
            class="MuiGridListTile-root"
            style="width: 16.666666666666668%; height: 138px; padding: 9px;"
          >
            <div
              class="MuiGridListTile-tile"
            >
              <img
                class="MuiCardMedia-root makeStyles-logo-36 MuiCardMedia-media MuiCardMedia-img"
                src="tumlogo.png"
              />
            </div>
          </li>
          <li
            class="MuiGridListTile-root"
            style="width: 16.666666666666668%; height: 138px; padding: 9px;"
          >
            <div
              class="MuiGridListTile-tile"
            >
              <img
                class="MuiCardMedia-root makeStyles-logo-36 MuiCardMedia-media MuiCardMedia-img"
                src="manage.png"
              />
            </div>
          </li>
          <li
            class="MuiGridListTile-root"
            style="width: 16.666666666666668%; height: 138px; padding: 9px;"
          >
            <div
              class="MuiGridListTile-tile"
            >
              <img
                class="MuiCardMedia-root makeStyles-logo-36 MuiCardMedia-media MuiCardMedia-img"
                src="tech-talents.png"
              />
            </div>
          </li>
          <li
            class="MuiGridListTile-root"
            style="width: 16.666666666666668%; height: 138px; padding: 9px;"
          >
            <div
              class="MuiGridListTile-tile"
            >
              <img
                class="MuiCardMedia-root makeStyles-logo-36 MuiCardMedia-media MuiCardMedia-img"
                src="start.png"
              />
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
